frontend/src/pages/Policy/policyContent/CrisisOverviewContent.tsx

```diff
@@ -1,33 +1,41 @@
-import HetTerm from '../../../styles/HetComponents/HetTerm'
+import HetTermUnderline from '../../../styles/HetComponents/HetTermUnderline'
 
 export const gvaFacts = [
-  {
-    content: (
-      <>
-        In 2023, Atlanta experienced no fewer than six mass shootings, with each
-        event tragically{' '}
-        <HetTerm>claiming six lives and injuring 22 others</HetTerm>, marking a
-        series of deliberate, targeted attacks that shook the community.
-      </>
-    ),
-  },
+	{
+		content: (
+			<>
+				In 2023, Atlanta experienced {' '}
+				<HetTermUnderline>no fewer than six mass shootings,</HetTermUnderline> with each
+				event tragically claiming six lives and injuring 22 others, marking a
+				series of deliberate, targeted attacks that shook the community.
+			</>
+		),
+	},
 
-  {
-    content: (
-      <>
-        As of April 2024, firearms have injured four children in Atlanta,
-        raising the total to <HetTerm>53 children injured</HetTerm> since 2021,
-        underscoring an urgent need for protective measures.
-      </>
-    ),
-  },
-  {
-    content: (
-      <>
-        Only four months into 2024, Georgia has already witnessed a staggering
-        toll of over{' '}
-        <HetTerm>92 lives lost to firearm-related incidents</HetTerm>.
-      </>
-    ),
-  },
+	{
+		content: (
+			<>
+				As of April 2024, firearms have injured four children in Atlanta,
+				raising the total to <HetTermUnderline>53 children injured</HetTermUnderline> since 2021,
+				underscoring an urgent need for protective measures.
+			</>
+		),
+	},
 ]
+
+export const rocketFoundationFacts = [
+	{
+		content: (
+			<>
+				Gun violence in the U.S. is a growing crisis, claiming over 47,000 lives in 2021 alone, with {' '}<HetTermUnderline>more than half being homicides</HetTermUnderline>. This epidemic disproportionately affects people of color, women, and children, making guns the leading cause of death among American youth.
```

Comment:
is there a reason to add `{' '}` before the HetTermUnderline? I feel like if it needs extra left padding we could add it to the component?